### PR TITLE
Fix chart name in section help

### PR DESCRIPTION
**Pull request by billm**
Minio Storage section help incorrectly states that it is for Nextcloud.

### CODE REVIEW

**Review by stavros-k — CHANGES_REQUESTED** [review]
Please only do changes under `library/ix-dev/`. The change you have currently made won't persist and will be overwritten!

Thanks

File: charts/minio/1.7.23/questions.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ groups:
   - name: "Minio Configuration"
     description: "Configure Minio credentials"
   - name: "Storage"
-    description: "Configure Storage for Nextcloud"
+    description: "Configure Storage for Minio"
```
Comment (stavros-k) [inline charts/minio/1.7.23/questions.yaml:9]:
```suggestion
    description: "Configure Storage for Nextcloud"
```

**Review by stavros-k — COMMENTED** [review]
Please update patch `version` in Chart.yaml  (eg +0.0.1)

Thanks

**Review by stavros-k — APPROVED** [review]

**Comment by stavros-k** [comment]
Thanks!

**Comment by billm** [comment]
Thanks for bearing with me on this!

**Review by Qubad786 — APPROVED** [review]
Thanks for the fix @billm, Approved 👍

**Pull request merged**